openedx/core/djangoapps/content/learning_sequences/api/tests/test_outlines.py

```diff
@@ -256,25 +291,28 @@ def get_details(self, at_time):
         student_details = get_user_course_outline_details(self.course_key, self.student, at_time)
         return staff_details, student_details
 
+    def get_sequence_keys(self, exclude=[]):
```

Comment:
Please don't use a mutable default arg (`exclude`). I know it won't bite in this particular circumstance, but a lot of newer developers see this pattern, don't realize the side-effects-over-multiple-calls implications, and copy it into their own code in ways that will bite them.